drivers/espi/espi_mchp_xec.c

```diff
@@ -113,107 +113,107 @@ enum mchp_smvw_regs {
- *  6h    | SMVW02 | HOST_RST_ACK | RCIN#        | SMI#      | SCI#        |
- *  7h    | MSVW02 | res          | res          | res       | HOS_RST_WARN|
- * ------------------------------------------------------------------------|
+ * --------------------------------------------------------------------------------|
+ *  2h    | MSVW00 | res             | SLP_S5#      | SLP_S4#   | SLP_S3#          |
+ *  3h    | MSVW01 | res             | OOB_RST_WARN | PLTRST#   | SUS_STAT#        |
+ *  4h    | SMVW00 | PME#            | WAKE#        | res       | OOB_RST_ACK      |
+ *  5h    | SMVW01 | TARGET_BOOT_STS | ERR_NONFATAL | ERR_FATAL | TARGET_BOOT_DONE |
+ *  6h    | SMVW02 | HOST_RST_ACK    | RCIN#        | SMI#      | SCI#             |
+ *  7h    | MSVW02 | res             | res          | res       | HOS_RST_WARN     |
+ * --------------------------------------------------------------------------------|
  * Platform specific virtual wires
- * ------------------------------------------------------------------------|
- *  40h   | SMVW03 | res          | res          | DNX_ACK   | SUS_ACK#    |
- *  41h   | MSVW03 | SLP_A#       | res          | SUS_PDNACK| SUS_WARN#   |
- *  42h   | MSVW04 | res          | res          | SLP_WLAN# | SLP_LAN#    |
- *  43h   | MSVW05 | generic      | generic      | generic   | generic     |
- *  44h   | MSVW06 | generic      | generic      | generic   | generic     |
- *  45h   | SMVW04 | generic      | generic      | generic   | generic     |
- *  46h   | SMVW05 | generic      | generic      | generic   | generic     |
- *  47h   | MSVW07 | res          | res          | res       | HOST_C10    |
- *  4Ah   | MSVW08 | res          | res          | DNX_WARN  | res         |
- *  50h   | SMVW06 | ESPI_OCB_3   | ESPI_OCB_2   | ESPI_OCB_1| ESPI_OCB_0  |
+ * --------------------------------------------------------------------------------|
+ *  40h   | SMVW03 | res             | res          | DNX_ACK   | SUS_ACK#         |
+ *  41h   | MSVW03 | SLP_A#          | res          | SUS_PDNACK| SUS_WARN#        |
+ *  42h   | MSVW04 | res             | res          | SLP_WLAN# | SLP_LAN#         |
+ *  43h   | MSVW05 | generic         | generic      | generic   | generic          |
+ *  44h   | MSVW06 | generic         | generic      | generic   | generic          |
+ *  45h   | SMVW04 | generic         | generic      | generic   | generic          |
+ *  46h   | SMVW05 | generic         | generic      | generic   | generic          |
+ *  47h   | MSVW07 | res             | res          | res       | HOST_C10         |
+ *  4Ah   | MSVW08 | res             | res          | DNX_WARN  | res              |
+ *  50h   | SMVW06 | ESPI_OCB_3      | ESPI_OCB_2   | ESPI_OCB_1| ESPI_OCB_0       |
  */
 
 static const struct xec_signal vw_tbl[] = {
 	/* MSVW00 */
 	[ESPI_VWIRE_SIGNAL_SLP_S3]        = {MCHP_MSVW00, ESPI_VWIRE_SRC_ID0,
-					     ESPI_MASTER_TO_SLAVE},
+					     ESPI_CONTROLLER_TO_TARGET},
```

Comment:
This commit should be squashed with the change the header file.  Otherwise this will break bi-sectability.

Reply:
Commits squashed